src/Curl/Curl.php

```diff
@@ -699,6 +699,25 @@ public function setConnectTimeout($seconds)
     }
 
     /**
+     * Set Cookie String
+     *
+     * @access public
+     * @param  $string
+     */
+    public function setCookieString($string)
+    {
+        if(preg_match_all('/\s*([^;=]+)=([^;]+)/i',$string,$matches) > 0){
+            if(isset($matches[1]) && isset($matches[2])){
+                if(count($matches[1]) === count($matches[2])){
+                    foreach ($matches[1] as $handle => $key) {
+                        $this->setCookie($key, $matches[2][$handle]);
+                    }
+                }
+            }
+        }
+    }
```

Comment:
I believe this could be simplified to:
```php
public function setCookieString($value) {
    return $this->setOpt(CURLOPT_COOKIE, $value);
}
````
Usage:
```php
$value = 'fruit=apple; colour=red';
$curl->setCookieString($value);
```